### DISCUSSION TITLE
Proper Noun should not be translated

### DISCUSSION THREAD

**Discussion by NargiT** [Dymerz/StarCitizen-Localization#19] [General]
This is related to a picture I saw in a french translation. 

"Green Glade Station" was translated in to "Station de clairière verte". Not only it does not respect the case but it's a bad practice. 

I see one big problem: It will create a gap between the internationnal community and local communities. People will not be able to understand what is happening as most people will use the original name to refer to it (even accronymes). This goes also with Organisation names or any brand. "Nintetails" must not become "Neuf queues".

![image](https://github.com/Dymerz/StarCitizen-Localization/assets/3153333/3e946bc3-48d6-406d-a0eb-159072f12c05)

**Comment by ROBdk97** [comment]
This probably just happened because of using translation services but we just did use such services to handle the bulk so the amount to translate would be less. Now everything is just about fine-tuning the results.

And yes I agree. Locations and such should not be translated.

**Comment by Dymerz** [comment]
I completely agree with you. This translation was generated by AI and is currently under active development by the French community. In the updated version, it seems that this issue has been fixed.

I'm working on integrating the French community into this repository, but it may take some time.

The [French repo](https://github.com/SPEED0U/StarCitizenTranslations)
